### ISSUE TITLE
Rowv/Colv and dendrogram

### ISSUE THREAD

**Issue by TriLe965** [talgalili/heatmaply#157]
I don't know if misunderstand the meaning of `Rowv/Colv` and `dendrogram`. I think there is something wrong.
```
p1 <- heatmaply(iris[, -5], Colv=T, Rowv=F, dendrogram="none", main="p1")
p2 <- heatmaply(iris[, -5], Colv=F, Rowv=F, dendrogram="none", main="p2")
p3 <- heatmaply(iris[, -5], Colv=T, Rowv=F, dendrogram="column", main="p3")
p4 <- heatmaply(iris[, -5], Colv=F, Rowv=F, dendrogram="column", main="p4")
```
I had no warning or anything wrong when running these lines. And this is the result

p1 - There is nothing wrong with it.
![image](https://user-images.githubusercontent.com/11041356/36478240-04cc6372-173f-11e8-8159-879059ef569e.png)
p2 - Shouldn't the column's order be the same as p3
![image](https://user-images.githubusercontent.com/11041356/36478246-0e99a784-173f-11e8-955e-d065e94fd29a.png)
p3 - There is nothing wrong with it
![image](https://user-images.githubusercontent.com/11041356/36478265-1bd82876-173f-11e8-976a-63473472eeb7.png)
p4 - It is ok too. But I think it needs a warning.
![image](https://user-images.githubusercontent.com/11041356/36478288-2fcbd8dc-173f-11e8-935c-cebf58b2c626.png)

**Comment by micdonato** [comment]
Kind of off-topic: why, in p3, there is an indicator of the grouping of the dendrogram? Is there a way to make that disappear?

**Comment by Alanocallaghan** [comment]
Hi, I think that colv and rowv decide whether a dendrogram is used for row/column ordering, not just if they are displayed. Maybe they should be aliases for a "show_column_dendrogram" option. Either way the documentation should reflect that

On 23 Apr 2018 11:40 pm, micdonato <notifications@github.com> wrote:

Kind of off-topic: why, in p3, there is an indicator of the grouping of the dendrogram? Is there a way to make that disappear?

**Comment by talgalili** [comment]
This should have been solved in #107 - if the doc is not clear enough please re-open and/or offer how to improve.
